.github/workflows/ci.yaml

```diff
@@ -517,7 +517,7 @@ jobs:
             env.HA_SHORT_VERSION }}-
       - name: Install additional OS dependencies
         if: steps.cache-venv.outputs.cache-hit != 'true'
-        timeout-minutes: 10
+        timeout-minutes: 15
```

Comment:
In CI run https://github.com/home-assistant/core/actions/runs/17585060425/job/49951309939?pr=150999, most of the successful steps completed in between 30s and 2m30s

One did finish in 9m25s, but I still believe that 10m is a good timeout as you have no idea if the 6 jobs that failed would have needed 11 minutes or 3 hours or 6 hours to complete